packages/ilmomasiina-backend/src/mail/config.ts

```diff
@@ -48,6 +48,8 @@ const mailTransporter: Transporter = (() => {
       });
       input.on('end', () => {
         console.log(data);
+        // temporary writefile
```

Comment:
```suggestion
        // writefile helper for development, testing styles
```
Add more descriptive comment